src/Core/Domain/Feature/FeatureValue/QueryResult/EditableFeatureValue.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\Feature\FeatureValue\QueryResult;
+
+use PrestaShop\PrestaShop\Core\Domain\Feature\ValueObject\FeatureId;
+use PrestaShop\PrestaShop\Core\Domain\Language\FeatureValueId;
+
+/**
+ * Stores data that's needed for editing.
+ */
+class EditableFeatureValue
+{
+    /**
+     * @var FeatureValueId
+     */
+    private $featureValueId;
+
+    /**
+     * @var FeatureId
+     */
+    private $featureId;
+
+    /**
+     * @var array
+     */
+    private $localizedValues;
+
+    /**
+     * @param FeatureValueId $featureValueId
+     * @param FeatureId $featureId
+     * @param array $localizedValues
+     */
+    public function __construct(FeatureValueId $featureValueId, FeatureId $featureId, array $localizedValues)
+    {
+        $this->featureValueId = $featureValueId;
+        $this->featureId = $featureId;
+        $this->localizedValues = $localizedValues;
+    }
+
+    /**
+     * @return FeatureValueId
+     */
+    public function getFeatureValueId()
+    {
+        return $this->featureValueId;
+    }
+
+    /**
+     * @return FeatureId
+     */
+    public function getFeatureId()
+    {
+        return $this->featureId;
+    }
+
+    /**
+     * @return array
+     */
+    public function getValue()
```

Comment:
Add comment here `Localized values indexed by language id` (or locale if it's indexed by locale)
Also rename this getter to  `getLocalizedValues` which instantly gives an hint to what it contains